index.ts

```diff
@@ -1247,6 +1394,7 @@ export const translations = {
         "Endergie gecompenseerde snelheid i.r.t. opdracht",
         "",
         "Velocità compensata dall'energia relativa al compito",
+        "",
```

Comment:
teSpeedDescription - Could you provide some article/wiki with definition what it is? It will help with translation.

Reply:
teSpeed and teSpeedDescription are currently not in use. Empty strings will fallback to english so this is okay for now. Thanks!